event_types.go

```diff
@@ -554,7 +586,10 @@ type MergeEvent struct {
 			Previous []Label `json:"previous"`
 			Current  []Label `json:"current"`
 		} `json:"labels"`
-		UpdatedByID []int `json:"updated_by_id"`
+		UpdatedByID struct {
+			Previous []Label `json:"previous"`
+			Current  []Label `json:"current"`
+		} `json:"updated_by_id"`
```

Comment:
This field was updated a few weeks ago in PR #593 to match how Gitlab uses this field. I'm guessing this is something that recently changed in the Gitlab API, so if you are using an older Gitlab version you may want to keep using v0.16.1 of this package instead of `master`?

Reply:
I will do a dedicted PR for this ;)